src/UKHO.ADDS.EFS.Orchestrator/Pipelines/Completion/Nodes/S57/CommitFileShareBatchNode.cs

```
@@ -28,11 +29,11 @@ protected override async Task<NodeResultStatus> PerformExecuteAsync(IExecutionCo
         {
             var job = context.Subject.Job!;
 
-            try
-            {
-                var commit = await _fileService.CommitBatchAsync(job.BatchId!, job.GetCorrelationId(), Environment.CancellationToken);
-            }
-            catch (Exception ex)
+            var batchHandle = new BatchHandle((string)job.BatchId!);
+
+            var commitBatchResult = await _fileService.CommitBatchAsync(batchHandle, (string)job.GetCorrelationId(), Environment.CancellationToken);
+
+            if (!commitBatchResult.IsSuccess(out _, out _))
```

Again, can just let the exception throw

---

Refactored code